.github/workflows/python-build-and-lint.yml

```diff
@@ -49,3 +49,8 @@ jobs:
           path: |
             python/docs/doc_artifacts/api
             python/docs/doc_artifacts/project_documentation
+
+      - name: Build PDF
```

Comment:
So, if you are going to include it here instead, which might be better location and leave the sphinx workflow to be solely for the website management, you'd still need to modify the artifact upload to include the output PDF.

Reply:
I was putting it in this file to make sure it was running fine in the pipeline, since the sphinx workflow doesn't run on pull requests.  Do you think it would be better here instead of sphinx.ymk?

Reply:
That caveat too would be another good reason to include with build and lint over the sphinx workflow.